pkg/apis/kops/componentconfig.go

```diff
@@ -34,6 +34,10 @@ type KubeletConfigSpec struct {
 	TLSCertFile string `json:"tlsCertFile,omitempty" flag:"tls-cert-file"`
 	// TODO: Remove unused TLSPrivateKeyFile
 	TLSPrivateKeyFile string `json:"tlsPrivateKeyFile,omitempty" flag:"tls-private-key-file"`
+	// TLSCipherSuites indicates the allowed TLS cipher suite
```

Comment:
So you need to add these fields to v1alpha1 and v1alpha2 (sorry, but that's the way the k8s apimachinery works).  If you search for TLSPrivateKeyFile you'll see the duplication.  Then `make apimachinery` will work :-)

Context: it's possible to change fields in different api versions, and then we would have to write custom conversion functions.  As we're adding the same field to all versions (which is safe) the autogenerated functions work.

Reply:
Done, the new fields are now present on `pkg/apis/kops/component.go`, `pkg/apis/kops/v1alpha1/componentconfig.go` and `pkg/apis/kops/v1alpha2/componentconfig.go`. Code generated by `make apimachinery` seems to be correct now.